### ISSUE TITLE
More Graph Layouts

### ISSUE THREAD

**Issue by illusion0001** [cirquit/trdrop#101]
Just a tracker for myself to implement this in the future. If anyone has ideas, comment below.

**Comment by cirquit** [comment]
Those are really nice ideas, there would be some code restructuring necessary to implement that, but I think it's perfectly feasible.

**Comment by illusion0001** [comment]
I also have a compact layout, would you like me to PR these? the text will need to be slightly aligned of course.
![](https://cdn.discordapp.com/attachments/543065295280144385/869463205557207040/unknown.png)

**Comment by illusion0001** [comment]
> 
> 
> Those are really nice ideas, there would be some code restructuring necessary to implement that, but I think it's perfectly feasible.

@cirquit yep, i found code for frametime data and the plot itself but they plot all in just a single graph and I don't know how I could separate it.

**Comment by cirquit** [comment]
In my current proposal for the v2 this should be easily possible!
